package.json

```diff
@@ -19,5 +33,13 @@
   "bugs": {
     "url": "https://github.com/plugblockchain/plug-api-types/issues"
   },
-  "homepage": "https://github.com/plugblockchain/plug-api-types#readme"
+  "homepage": "https://github.com/plugblockchain/plug-api-types#readme",
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged --pattern 'types/*' && npx tslint -c tslint.json --project ."
```

Comment:
Have you tried with writing some unformatted code, and see if the tools help with automatic formatting?

Reply:
yes